integration_test.go

```diff
@@ -3039,3 +3039,196 @@ func consumeScanner(c gohbase.Client, scan *hrpc.Scan) ([]*hrpc.Result, error) {
 
 	return rs, nil
 }
+
+func TestReplication(t *testing.T) {
+	ctx := context.Background()
+	ac := gohbase.NewAdminClient(*host)
+	peerIdPrefix := table
+	peerIdRegex := peerIdPrefix + ".*"
+
+	t.Run("check no pre-existing peers for test table",
+		func(t *testing.T) {
+			listPeers, err := hrpc.NewListReplicationPeers(ctx,
+				hrpc.ListReplicationPeersRegex(peerIdRegex))
+			if err != nil {
+				t.Fatal(err)
+			}
+			peers, err := ac.ListReplicationPeers(listPeers)
+			if err != nil {
+				t.Fatal(err)
+			}
+			if len(peers) != 0 {
+				t.Fatalf("Expected 0 peers. Got %d", len(peers))
+			}
+		})
+
+	// Cleanup after test
+	defer func() {
+		listPeers, err := hrpc.NewListReplicationPeers(ctx,
+			hrpc.ListReplicationPeersRegex(peerIdRegex))
+		if err != nil {
+			t.Fatal(err)
+		}
+		peers, err := ac.ListReplicationPeers(listPeers)
+		if err != nil {
+			t.Fatal(err)
+		}
+		for _, peer := range peers {
+			removePeer := hrpc.NewRemoveReplicationPeer(ctx, peer.GetId())
+			err := ac.RemoveReplicationPeer(removePeer)
+			if err != nil {
+				t.Fatal(err)
+			}
+		}
+	}()
+
+	tcases := []struct {
+		serial  bool
+		enabled bool
+	}{
+		{
+			serial:  false,
+			enabled: true,
+		},
+		{
+			serial:  false,
+			enabled: false,
+		},
+		{
+			serial:  true,
+			enabled: true,
+		},
+		{
+			serial:  true,
+			enabled: false,
+		},
+	}
+	for i, tc := range tcases {
+		testNoStr := strconv.Itoa(i)
+		t.Run("create list and invert enabled state of peer test #"+testNoStr, func(t *testing.T) {
+			peerId := peerIdPrefix + "_" + testNoStr
+			clusterKey := strings.Join([]string{
+				*host, "2181",
+				"/hbase-" + testNoStr,
+			}, ":")
+			tableBytes := []byte(table + "_" + testNoStr)
```

Comment:
Does the table not have to match the name of the actual table for this to work (re: adding the `testNoStr` suffix)?